.github/workflows/stream-connectors-dependencies.yml

```diff
@@ -79,19 +79,40 @@ jobs:
         working-directory: dependencies/${{ matrix.lib }}
         shell: bash
 
+      - if: ${{ matrix.lib == 'lua-curl' }}
```

Comment:
```suggestion
      - if: matrix.lib == 'lua-curl'
```

Reply:
using name could help while debugging the logs ;)

Reply:
this is currently the main syntax we use in the workflows : https://github.com/centreon/centreon/blob/develop/.github/workflows/web.yml#L95
I think it's better to let it for consistency

Reply:
in this case, ${{}} are not required as it is a string
adding a name to the step, helps while reading the logs

Reply:
indeed I will add name